server/util/clickhouse/schema/schema.go

```diff
@@ -108,6 +109,7 @@ type Invocation struct {
 	TotalUncachedActionExecUsec       int64
 	DownloadThroughputBytesPerSecond  int64
 	UploadThroughputBytesPerSecond    int64
+	Tags                              pq.StringArray `gorm:"type:Array(String);"`
```

Comment:
isn't pq a Postgres driver?

Reply:
I somehow convinced myself that this was a universal driver, I'm going to hack on making one for Clickhouse for a bit because there doesn't seem to be one.

Reply:
This is going to take me a bit, so I'm gonna shelve this for a day or two and get the customer value UI stuff out the door first.